include/yaramod/types/modules/module.h

```diff
@@ -9,13 +9,14 @@
 #include <memory>
 #include <string>
 
+#include "yaramod/utils/json.h"
 #include "yaramod/types/symbols.h"
+#include "yaramod/yaramod_error.h"
 
 namespace yaramod {
 
 /**
- * Class representing features of module.
- * Every module has to provide its features to state what must hold for it to be loaded.
+ * Class representing features of Yaramod.
  */
 enum Features
```

Comment:
TODO: Move `Features` class to other file. The `Module` class no longer depends on `Features` class

Reply:
Done